server/routes/static.py

```diff
@@ -42,13 +40,20 @@ def about():
     env_name = current_app.config.get('ENV_NAME', None)
```

Comment:
move `env_name = current_app.config.get('ENV_NAME', None)` to the top of this module and only declare once?

Reply:
It requires an application context to load the value, so it won't work at the top of the module. Instead, I've added it to the request global. ptal and lmk what you think

Reply:
This is concise!